dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -288,7 +288,10 @@ async def _setup_runtime_env(
             # TODO(chenk008): Add log about allocated_resource to
             # avoid lint error. That will be moved to cgroup plugin.
             per_job_logger.debug(f"Worker has resource :" f"{allocated_resource}")
-            context = RuntimeEnvContext(env_vars=runtime_env.env_vars())
+            context = RuntimeEnvContext(
+                env_vars=runtime_env.env_vars(),
+                julia_command=runtime_env.julia_command(),
```

Comment:
add TODO here to use plugin instead
```suggestion
                # TODO: use plugin instead of special casing
                julia_command=runtime_env.julia_command(),
```